src/idsk/components/footer-extended/template.njk

```diff
@@ -148,6 +150,7 @@
                                     }
                                 }) }}
                                                 </div>
+                                                <div id="feedback-info" data-email="{{ params.feedbackEmail if params.feedbackEmail else 'idsk@vicepremier.gov.sk' }}" data-subject="{{ params.feedbackSubject if params.feedbackSubject else 'Nahlásenie chyby'}}">{{ params.feedbackMessage if params.feedbackMessage else 'Dobrý deň, na stránke idsk.gov.sk máte problém týkajúci sa: issue. Popis chyby: description' }}</div>
```

Comment:
- ideckam by som sa vyvaroval,
- description a issue moze nastat v inom jazyku ze sa bude nachadzat aj v texte, riesil by som to asi niecim takym ako %description% a %issue% aby sme sa vyhli niecomu takemu